dev/sg/internal/run/run.go

```diff
@@ -92,7 +119,7 @@ func Commands(ctx context.Context, globalEnv map[string]string, verbose bool, cm
 	}
 }
 
-func waitForInstallation(cmdNames map[string]struct{}, installed chan string, failures chan failedRun, okayToStart chan struct{}) error {
+func waitForInstallation(cmdNames map[string]struct{}, installed chan string, failures chan failedRun, okayToStart chan struct{}, postInstallCallback func() error) error {
```

Comment:
It's obvious that it's a callback :)

```suggestion
func waitForInstallation(cmdNames map[string]struct{}, installed chan string, failures chan failedRun, okayToStart chan struct{}, postInstall func() error) error {
```